hack/jenkins/test-flake-chart/flake_chart.js

```diff
@@ -664,30 +665,39 @@ function displayEnvironmentChart(testData, environmentName) {
 }
 
 async function init() {
+  const query = parseUrlQuery(window.location.search);
+  const desiredTest = query.test, desiredEnvironment = query.env || "", desiredPeriod = query.period || "";
+
   google.charts.load('current', { 'packages': ['corechart'] });
   let testData, responseDate;
   try {
     // Wait for Google Charts to load, and for test data to load.
     // Only store the test data (at index 1) into `testData`.
     [testData, responseDate] = (await Promise.all([
       new Promise(resolve => google.charts.setOnLoadCallback(resolve)),
-      loadTestData()
+      loadTestData(desiredPeriod)
     ]))[1];
   } catch (err) {
     displayError(err);
     return;
   }
 
-  const query = parseUrlQuery(window.location.search);
-  const desiredTest = query.test, desiredEnvironment = query.env || "";
-
   if (desiredTest === undefined) {
     displayEnvironmentChart(testData, desiredEnvironment);
   } else {
     displayTestAndEnvironmentChart(testData, desiredTest, desiredEnvironment);
   }
   document.querySelector('#data_date_container').style.display = 'block';
   document.querySelector('#data_date').innerText = responseDate.toLocaleString();
+  let periodDisplay, newURL;
+  if (desiredPeriod === 'last90') {
+    newURL = window.location.href.replace(/&?period=last90/gi, '');
```

Comment:
should these conditions be flipped?

Reply:
This is creating the link to the opposite page, `desiredPeriod` is the current page, so I'm stripping the current link for the all data link that's displayed on the top of the page if that makes sense.

Reply:
ok I get it. a little confusing so maybe a comment is in order.

otherwise lgtm